src/CSET/operators/filters.py

```diff
@@ -21,30 +21,30 @@
 
 
 def filter_cubes(
-    cubelist: iris.cube.CubeList, constraint: iris.Constraint, **kwargs
+    cube: iris.cube.Cube, constraint: iris.Constraint, **kwargs
 ) -> iris.cube.Cube:
     """
     Filters a cubelist down to a single cube based on a constraint.
 
     Arguments
     ---------
-    cubelist: iris.cube.CubeList
-        Cubes to iterate over
+    cube: iris.cube.Cube | iris.cube.CubeList
+        Cube(s) to iterate over
     constraint: iris.Constraint
         Constraint to extract
 
     Returns
     -------
-    cube: iris.cube.Cube
-        Single variable
+    filtered_cube: iris.cube.Cube
+        Filtered cube
 
     Raises
     ------
     ValueError
         If the constraint doesn't produce a single cube.
     """
 
-    filtered_cubes = cubelist.extract(constraint)
+    filtered_cubes = cube.extract(constraint)
 
     # Check filtered cubes is a CubeList containing one cube.
     if len(filtered_cubes) == 1:
```

Comment:
I need to think about this. If ensemble data, and there is no realization dimension, it will probably return a list of cubes with near identical metadata (with maybe some attribute str mentioning which member it is). It looks like `_is_ensemble` returns `True` if there is a realization dimension, or multiple cubes with identical xml. Won't L50 here prevent multiple cubes being passed any further though (i.e. a `iris.cube.CubeList`).